app/tasks/doctor.php

```diff
@@ -9,11 +9,15 @@
 use Utopia\App;
 use Utopia\CLI\Console;
 use Utopia\Domains\Domain;
+use Utopia\Validator\Text;
+use Utopia\Validator\Wildcard;
 
 $cli
     ->task('doctor')
     ->desc('Validate server health')
-    ->action(function () use ($register) {
+    ->param('interactive', 'n', new Text(1), 'Run an interactive session', true)
+    ->param('receiver', '', new Wildcard(), 'Send an email to this address to test SMTP', true)
```

Comment:
We have a [validator for email](https://github.com/appwrite/appwrite/blob/master/src/Appwrite/Network/Validator/Email.php) so let's use that instead of wildcard